test/TimeoutTest.php

```diff
@@ -8,6 +8,7 @@
 use Amp\Pause;
 use Amp\Success;
 use Amp\Promise;
+use function React\Promise\resolve;
```

Comment:
Fine with me either way, that's why it's a question. We don't have `resolve` anymore anyway.